### DISCUSSION TITLE
It's been too long... Upscayl v2.15 is COMING SOON! 😄

### DISCUSSION THREAD

**Discussion by NayamAmarshe** [upscayl/upscayl#1067] [Announcements]
Hello to the one reading this!

<img width="1680" alt="Screenshot 2024-12-15 at 23 30 40" src="https://github.com/user-attachments/assets/99bd0b22-4fb3-4e03-9c70-132697db76a7" />

I'm sorry for disappearing for so long. Upscayl hasn't had an update in 6 months and it's mostly due to the big features that we have introduced and also because I have been busy with my job, Upscayl Cloud and whatever life I can fit in-between these things. So I apologize for the delay.

Good news is, I'm going to launch Upscayl v2.15 very soon! This new release will come with some interface updates, more languages and a really cool feature that allows you to paste images from clipboard (yay, finally!).

Thanks for all the support. I'll be releasing the update in the next 2 days (if I don't get busy again 😅).

**Comment by andreybexus** [comment]
I hope everything will be fine with the creative upscale technology? Will there be any improvements in the upscale itself? I think in the future there will be cool effects such as HDR, clarity, Hi-Vision, UHD and others.

**Comment by andreybexus** [comment]
And when is a full-fledged release planned?

**Reply by NayamAmarshe** [reply]
Next week :)

**Reply by andreybexus** [reply]
How will the release differ from the pre-release?

**Reply by andreybexus** [reply]
> Next week :)

It would be cool if you could make Upscale videos and audio, thereby improving their quality, and scale the bitrate and frequencies in audio.

**Reply by NayamAmarshe** [reply]
> How will the release differ from the pre-release?

Bug fixes and a few additions

**Reply by andreybexus** [reply]
> > How will the release differ from the pre-release?
> 
> Bug fixes and a few additions
I hope the quality of the upscale on the release will improve so that you can see the details.

**Comment by andreybexus** [comment]
What has changed in version 2.15.1?

**Reply by andreybexus** [reply]
I would like the TAA mode to work faster.

**Comment by andreybexus** [comment]
In Upcayl Lite, there are no artifacts, but in the standard model they are. Please, I hope you will fix this in the next version. Artifacts are cubes when you zoom in on the image.

**Comment by ultraderek** [comment]
I'm the guy that read your flatpak notes

**Reply by aaronliu0130** [reply]
nayam's release notes are for everyone! :heart:

**Comment by Tprodt** [comment]
Let me say on behalf of most of us here that you my man are the GOAT. Thank you very much for this priceless app
